app/src/test/java/co/nimblehq/compose/crypto/ui/screens/home/HomeViewModelTest.kt

```diff
@@ -0,0 +1,72 @@
+package co.nimblehq.compose.crypto.ui.screens.home
+
+import app.cash.turbine.test
+import co.nimblehq.compose.crypto.domain.usecase.GetMyCoinsUseCase
+import co.nimblehq.compose.crypto.test.MockUtil
+import co.nimblehq.compose.crypto.ui.screens.BaseViewModelTest
+import co.nimblehq.compose.crypto.ui.uimodel.toUiModel
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.*
+
+@ExperimentalCoroutinesApi
+class HomeViewModelTest : BaseViewModelTest() {
+
+    private val mockGetMyCoinsUseCase = mockk<GetMyCoinsUseCase>()
+    private lateinit var viewModel: HomeViewModel
+
+    @Before
+    fun setUp() {
+        every { mockGetMyCoinsUseCase.execute(any()) } returns flowOf(MockUtil.myCoins)
+
+        Dispatchers.setMain(testDispatcher)
+    }
+
+    @After
+    fun tearDown() {
+        Dispatchers.resetMain()
+        testDispatcher.cleanupTestCoroutines()
+    }
+
+    @Test
+    fun `When getting my coin list successfully, it should return my coin list`() =
+        runBlockingTest {
+            testDispatcher.pauseDispatcher()
```

Comment:
It seems `deprecated` here. Did you get like this 👇  on your side @luongvo 
![Screen Shot 2565-09-06 at 19 04 36](https://user-images.githubusercontent.com/28002315/188631257-8bcd3694-fff1-448c-bd4c-cbe4e23b3ed9.png)

Reply:
@Wadeewee As I described here https://github.com/nimblehq/jetpack-compose-crypto/pull/27#discussion_r962587131, I will resolve it in next PRs.